packages/ui/src/components/node-group.tsx

```diff
@@ -33,10 +33,11 @@ function Icon({
     <div className="col-start-1 row-start-1 self-start">
       <div
         className={cn(
-          'border-tertiary-background/30 bg-background text-primary relative z-20 flex h-6 w-6 place-content-center place-items-center rounded-full border p-1',
+          'border-tertiary-background/30 bg-background text-primary relative flex h-6 w-6 place-content-center place-items-center rounded-full border p-1',
           { 'bg-transparent border-none size-1 p-0 mt-2 shadow-commit-list-bullet': simpleNodeIcon },
           className
         )}
+        style={{ zIndex: 15 }}
```

Comment:
avoid inline styles. better options in order of priority:
1. lets check with @knagurski if we can enable tailwind in gitness app
2. create semantic classes like `layer-low`, `layer-medium`, `layer-high` to avoid proliferation of 20 different z layers.
3. create a specific class for this style in a local css file

Reply:
Any reason to not just use something like `z-20`? `15` seems oddly specific 🤔

Reply:
@knagurski z-index 20 is being used by Harness's sidenav module drawer. So, we have to use z-index values less than 20 for all elements in MFE to prevent overlapping.

Reply:
Would `z-10` work instead?